pkgs/by-name/ma/maltego/package.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchzip
+, jre
+, giflib
+, gawk
+, makeBinaryWrapper
+, icoutils
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+stdenv.mkDerivation rec {
+  pname = "maltego";
+  version = "4.6.0";
+  src = fetchzip {
+    url = "https://downloads.maltego.com/maltego-v4/linux/Maltego.v${version}.linux.zip";
+    hash = "sha256-q+1RYToZtBxAIDSiUWf3i/3GBBDwh6NWteHiK4VM1HY=";
+  };
+
+  postPatch = ''
+      substituteInPlace bin/maltego \
+            --replace /usr/bin/awk ${gawk}/bin/awk
+  '';
+
+    desktopItems = [
+    (makeDesktopItem {
+      name = pname;
+      desktopName = "Maltego";
+      exec = pname;
+      icon = pname;
+      comment = "An open source intelligence and forensics application";
+      categories = [ "Network" "Security" ];
+      startupNotify = false;
+    })
+  ];
+
+  nativeBuildInputs = [
+    icoutils
+    makeBinaryWrapper
+    copyDesktopItems
+  ];
+
+  buildInputs = [ jre giflib ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,share}
+    chmod +x bin/maltego
+    cp -aR . "$out/share/maltego/"
+
+    icotool -x bin/maltego.ico
+    for size in 16 32 48 256
+    do
+      mkdir -p $out/share/icons/hicolor/$size\x$size/apps
+      cp maltego_*_$size\x$size\x32.png $out/share/icons/hicolor/$size\x$size/apps/maltego.png
+    done
+
+    makeWrapper $out/share/maltego/bin/maltego $out/bin/${pname} \
+      --set JAVA_HOME ${jre} \
+      --prefix PATH : ${lib.makeBinPath [ jre ]}
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://www.maltego.com";
+    description = "An open source intelligence and forensics application, enabling to easily gather information about DNS, domains, IP addresses, websites, persons, and so on";
+    maintainers = with maintainers; [ emilytrau d3vil0p3r ];
```

Comment:
did you ask @emilytrau if she is fine being added as a maintainer?

Reply:
On 2022 @emilytrau opened a PR https://github.com/NixOS/nixpkgs/pull/185046 for 4.3.0 (I was aware of this after I created this PR, sorry) and I decided to mix her PR content with the current one (4.6.0), for this reason I added her.